src/Model/ModelManager.php

```diff
@@ -279,6 +279,10 @@ public function getNormalizedIdentifier($document)
             throw new \RunTimeException('Invalid argument, object or null required');
         }
 
+        // not managed
+        if (!$this->registry->getManagerForClass(\get_class($document))) {
+            return;
+        }
```

Comment:
Isn't that exactly what the lines below are supposed to be about?

Reply:
It's supposed to, but it doesn't.

Reply:
Maybe the condition you added should be added directly inside the condition line 287? Or maybe the issue is calling `getNormalizedIdentifier` with a collection, I don't think that it makes sense, does it?

Reply:
If you could provide a stack trace with the error you are getting, we might be able to figure out why this is happening.